src/NzbDrone.Core/Applications/ApplicationBase.cs

```diff
@@ -14,7 +14,7 @@ public abstract class ApplicationBase<TSettings> : IApplication
         protected readonly IAppIndexerMapService _appIndexerMapService;
         protected readonly Logger _logger;
 
-        protected static readonly Regex AppIndexerRegex = new Regex(@"\/(?<indexer>\d.)\/?$",
+        protected static readonly Regex AppIndexerRegex = new Regex(@"\/(?<indexer>\d{1,3})(?:\/(?:api)?\/?)?$",
```

Comment:
Why not `\/(?<indexer>\d*)(?:\/(?:api)?\/?)?$`

Just incase someone has 1000 :D

Reply:
If they have 1000+ then they'll have other issues than indexers not mapping 😂